tests/org.jboss.tools.cdi.bot.test/src/org/jboss/tools/cdi/bot/test/wizard/template/CDIWebProjectWizardTemplate.java

```diff
@@ -110,13 +96,21 @@ public void createCDIProjectWithoutBeansXml(){
 		}
 		assertEquals(sr.getRuntimeName(),fp.getTargetRuntime());
 		assertEquals("Dynamic Web Project with CDI "+CDIVersion+" (Contexts and Dependency Injection)",fp.getConfiguration());
-		fp.activateFacet("1.8", "Java");
+		if (isJavaLE8() || CDIVersion.equals("1.0")) {
+			fp.activateFacet("1.8", "Java");
+		} else {
+			fp.activateFacet("11", "Java");
+		}
 		cw.next();
 		cw.next();
 		cw.next();
 		CDIInstallWizardPage ip = new CDIInstallWizardPage(cw);
 		ip.toggleCreateBeansXml(false);
 		cw.finish();
+		new WaitUntil(new JobIsRunning(), TimePeriod.DEFAULT, false);
+		new WaitWhile(new JobIsRunning(), TimePeriod.LONG);
+		setWorkspaceDefaultJREForJava8();
```

Comment:
Why are you setting workspace default JRE in the middle of the test?

There could be used setDefault parameter in JRE requirement.

Reply:
Well, I have removed it. This statement has its meaning when I have not completed the configuration of Java 8 + 11.